turbo-txpool/src/lib.rs

```diff
@@ -1134,10 +1113,13 @@ mod tests {
         assert_eq!(inserted, [ImportResult::Success]);
 
         assert_eq!(inner.txs.len(), 1);
-        assert!(matches!(inner.txs[0], PooledTx {
-            runnable: false,
-            ..
-        }));
+        assert!(matches!(
+            inner.txs[0],
+            PooledTx {
+                runnable: false,
+                ..
```

Comment:
Does this mean "I don't care about the rest of the object"? Seems pretty sweet.

Reply:
Yep, exactly!